config/extra/with-perf.mk

```diff
@@ -0,0 +1,2 @@
+CPPFLAGS+=-fno-omit-frame-pointer -fno-inline
```

Comment:
fno-omit-frame-pointer is defaulted on for perf on all builds, I believe? I made that change.

Reply:
i see it for some other things but not for perf in specific. This is just for explicitly perf profiling.